### DISCUSSION TITLE
Pin docker containers used in wp-env to limit download/build size

### DISCUSSION THREAD

**Discussion by gorm** [WordPress/gutenberg#59173] [Developer Experience]
Hi,

I develop on a metered internet connection. Starting wp-env in a local WordPress installation. But wp-env seems to download a lot of MBs and recreate the docker installation quite often. So I wonder if it's somehow possible to pin wp-env to a specific version instead of relaying on latest docker images so that I do not download new versions every week?

I have tried to pin the php version and also running on a local static wordpress installation so don't quite understand why containers gets downloaded and rebuilt so often.

Thanks!

**Comment by talldan** [comment]
Not sure what it would be downloading. I believe it uses the latest production release of WordPress by default, so if a new release happens it'll re-download, but otherwise should use the locally cached version.

You can pin it to a specific version of WordPress by changing the 'core' setting in the .wp-env.json file, there's some examples in the documentation:
https://github.com/WordPress/gutenberg/tree/trunk/packages/env#examples

**Reply by gorm** [reply]
Ok, I tried to pin it to the local directory (as I run it from root of a WordPress installation), so I set it to `core: "./"`, but I guess this doesn't work like I expected so could be the issue. Will try to pin it to a docker version instead. Thanks!

**Reply by gorm** [reply]
It's been working well! After pinning to a specific version instead of "pinning" to local directory, I have not seen regular updates. Thanks Daniel!
